taker-frontend/src/App.tsx

```diff
@@ -120,6 +121,14 @@ export const App = () => {
     const cfdsOrUndefined = useLatestEvent<Cfd[]>(source, "cfds", intoCfd);
     let cfds = cfdsOrUndefined ? cfdsOrUndefined! : [];
     const connectedToMakerOrUndefined = useLatestEvent<ConnectionStatus>(source, "maker_status");
+    const makerCompatibilityOrUndefined = useLatestEvent<MakerCompatibility>(source, "maker_compatibility");
+
+    let incompatible = false;
+    if (makerCompatibilityOrUndefined) {
+        incompatible = makerCompatibilityOrUndefined.unsupported_protocols !== undefined
+            && makerCompatibilityOrUndefined.unsupported_protocols.length > 0;
+    }
```

Comment:
nit: this could have been a single expression instead of an assignment and if block 🙃